download.py

```diff
@@ -40,91 +48,112 @@
 
 WEBRTCVAD_SUPPORTED_SAMPLE_RATES_HZ = [8000, 16000, 32000, 48000]
 
-def download_csv_dataset():
-    print("Downloading CSV from", args.csv_url)
-    with requests.Session() as s:
-        download = s.get(args.csv_url)
+def does_cached_csv_dataset_exist(path_to_dataset_csv):
```

Comment:
Should we move this + several of the following functions to a "dataset processing" module of its own? All of the methods that deal with checking/downloading/reading the dataset?

Reply:
We have one and this function was an oversight, so I'll move it there.